src/mlpack/tests/lsh_test.cpp

```diff
@@ -466,6 +466,145 @@ BOOST_AUTO_TEST_CASE(DeterministicNoMerge)
   }
 }
 
+/**
+ * Test: Create an LSHSearch object and use an increasing number of probes to
+ * search for points. Require that recall for the same object doesn't decrease
+ * with increasing number of probes. Also require that at least a few times
+ * there's some increase in recall.
+ */
+BOOST_AUTO_TEST_CASE(MultiprobeTest)
+{
+  const double epsilonIncrease = 0.05;
+  const size_t repetitions = 5; // train five objects
+
+  const size_t probeTrials = 5;
+  const size_t numProbes[probeTrials] = {0, 1, 2, 3, 4};
+
+
+  /// algorithm parameters
+  const int k = 4;
+  const int numTables = 16;
+  const int numProj = 3;
+  const double hashWidth = 0;
+  const int secondHashSize = 99901;
+  const int bucketSize = 500;
+
+  const string trainSet = "iris_train.csv";
+  const string testSet = "iris_test.csv";
+  arma::mat rdata;
+  arma::mat qdata;
+  data::Load(trainSet, rdata, true);
+  data::Load(testSet, qdata, true);
+
+  // Run classic knn on reference set
+  KNN knn(rdata);
+  arma::Mat<size_t> groundTruth;
+  arma::mat groundDistances;
+  knn.Search(qdata, k, groundTruth, groundDistances);
+  
+  bool foundIncrease = 0;
+
+  for (size_t rep = 0; rep < repetitions; ++rep)
+  {
+    // train a model
+    LSHSearch<> multiprobeTest(
+        rdata,
+        numProj,
+        numTables,
+        hashWidth,
+        secondHashSize,
+        bucketSize);
```

Comment:
You could probably fit this on just one or two lines, but it does not matter to me.  Also the comment could be capitalized and have a period after it. :)